include/deal.II/multigrid/mg_transfer.templates.h

```diff
@@ -210,6 +210,8 @@ MGLevelGlobalTransfer<VectorType>::copy_to_mg(
   MGLevelObject<VectorType> &      dst,
   const InVector &                 src) const
 {
+  Assert(sizes.size() == dof_handler.get_triangulation().n_levels(),
```

Comment:
I found another bug in here (and in all other cases below):
```suggestion
  Assert(sizes.size() == dof_handler.get_triangulation().n_global_levels(),
```

Reply:
@kronbichler This does not seem to work either (see CI).